jobserver/settings.py

```diff
@@ -174,6 +173,19 @@
 }
 
 
+# Checks
+# https://docs.djangoproject.com/en/3.2/topics/checks/
+SILENCED_SYSTEM_CHECKS = [
+    "security.W004",  # (SECURE_HSTS_SECONDS) TLS is handled by Nginx
+    "security.W008",  # (SECURE_SSL_REDIRECT) HTTPS redirection is handled by CloudFlare
+]
+
+# Security
+# https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-CSRF_COOKIE_SECURE
+CSRF_COOKIE_SECURE = not DEBUG
+SESSION_COOKIE_SECURE = not DEBUG
```

Comment:
Hmm, so DEBUG is entirely possibly to set in prod, right?

Does `manage.py check --deploy` error if DEBUG is set?

Reply:
> DEBUG is entirely possibly to set in prod, right?

Yes

> Does manage.py check --deploy error if DEBUG is set?

No it raises a warning, which doesn't cause `manage.py check --deploy` to return a failed error code.